src/components/ResultsItem/ResultsItem.css

```diff
@@ -48,4 +48,19 @@
 
 .results-item-footer {
     font-size: 12px;
-}
\ No newline at end of file
+}
+
+#parent{position: relative;}
+
+.hover-div{
+    z-index: 9;
+}
+
+.hover-video{
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    top: 40px;
```

Comment:
from testing different values for top and left from my end, i think it looks nicer when `top: 30px;` and `left: 0px;` since that way the video shows right under the link! feel free to lmk if you think otherwise though